composer.json

```diff
@@ -15,10 +15,11 @@
         }
     ],
     "require": {
-        "php": "^7.0"
+        "php": "^7.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.2"
+        "phpunit/phpunit": "^6.2",
+        "symfony/process": "^2 || ^3.3 || ^4"
```

Comment:
What is `symfony/process` used for?

Reply:
Nevermind, it's used for an interesting take on integration tests :-)

Reply:
It is used in the integration tests only to check if the generated `diff`'s are correct, using the commands `git apply` and `patch` on Linux based test environments.